3478/1/top_gems/github_parser.rb

```diff
@@ -0,0 +1,53 @@
+require 'octokit'
+require 'open-uri'
```

Comment:
Where is Gemfile? I am not a psychic, and I can not Know, what gems are needed to run. All dependencies must be in Gemfile.

Reply:
Not sure if i'm doing it in a right way